packages/advisor-components/src/RuleDetails/RuleDetails.scss

```
@@ -0,0 +1,91 @@
+@import '~@patternfly/patternfly/sass-utilities/_all';
+
+hr {
+  border: 0.01rem solid #ccc;
+  margin: var(--pf-global--spacer--md) 0 var(--pf-global--spacer--md) 0;
+}
+
+.ins-c-rule-details__split {
+  display: inline-flex;
+  justify-content: space-around;
+  .pf-l-split__item {
+    position: relative;
+  }
+}
+
+.ins-c-rule-details__stack {
+  width: 600px;
+
+  .ins-c-rule-details__stackitem {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    .pf-c-content {
+      font-size: var(--pf-global--FontSize--sm);
+    }
+
+    .description-stack-override {
```

Can we use `ins-c-*` prefix here?
```suggestion
    .ins-c-description-stack-override {
```